### PR TITLE
Adding `oauth_allow_insecure_email_lookup` to fix oauth user sync error

### PR DESCRIPTION
**Pull request by hypery2k**
Adding this options helps to solve this issue #74154

### CODE REVIEW

**Comment by patcher99** [comment]
fyi @gardar, Since you are also working on the grafana role

**Comment by ishanjainn** [comment]
fyi @gardar as you are also working in the grafana role

**Review by ishanjainn — APPROVED** [review]

File: roles/grafana/templates/grafana.ini.j2 [hunk 1]
```diff
@@ -71,7 +71,8 @@ welcome_email_on_sign_up = {{ grafana_welcome_email_on_sign_up }}
 {% if grafana_auth != {} %}
 [auth]
 disable_login_form = {{ grafana_auth.disable_login_form | default('False') }}
-oauth_auto_login = {{ grafana_auth.oauth_auto_login | default('False') }}
+oauth_allow_insecure_email_lookup = {{ grafana_auth.oauth_allow_insecure_email_lookup | default('False') }}
+disable_signout_menu = {{ grafana_auth.disable_signout_menu | default('False') }}
```
Comment (gardar) [inline roles/grafana/templates/grafana.ini.j2:75]: Not sure why you are duplicating the line
```
disable_signout_menu = {{ grafana_auth.disable_signout_menu | default('False') }}
```

Also not sure why you are removing 
```
oauth_auto_login = {{ grafana_auth.oauth_auto_login | default('False') }
```

**Review by gardar — CHANGES_REQUESTED** [review]

File: roles/grafana/templates/grafana.ini.j2 (hunk 1 above)
Comment (hypery2k) [reply roles/grafana/templates/grafana.ini.j2]: sorry, my fault on rebasing, fixed that.

Thanks for the hint

**Review by gardar — APPROVED** [review]

**Pull request merged**